### PR TITLE
release prep: build .zip needed for test execution

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

File: tests/src/test/resources/without_dir_entries/main.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def main(params = {})
```
Comment (rabbah) [inline tests/src/test/resources/without_dir_entries/main.rb]: No license?

File: tests/src/test/resources/without_dir_entries/bundle/bundler/setup.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'rbconfig'
+# ruby 1.8.7 doesn't define RUBY_ENGINE
```
Comment (rabbah) [inline tests/src/test/resources/without_dir_entries/bundle/bundler/setup.rb]: No license?

Comment (dgrove-oss) [reply tests/src/test/resources/without_dir_entries/bundle/bundler/setup.rb]: good catch; did header PR separate from this one.  Fixed.

File: tests/src/test/resources/without_dir_entries/main.rb (hunk 1 above)
Comment (dgrove-oss) [reply tests/src/test/resources/without_dir_entries/main.rb]: fixed

**Pull request merged**